content/en/tracing/setup/dotnet-framework.md

```diff
@@ -182,7 +188,9 @@ To configure the Tracer using an `app.config` or `web.config` file, use the `<ap
 <configuration>
   <appSettings>
     <add key="DD_TRACE_AGENT_URL" value="http://localhost:8126"/>
-    <add key="DD_SERVICE_NAME" value="MyService"/>
+    <add key="DD_ENV" value="prod"/>
+    <add key="DD_SERVICE" value="MyService"/>
+    <add key="DD_VERSION" value="abc123"/>
     <add key="DD_ADONET_ENABLED" value="false"/>
```

Comment:
```suggestion
    <add key="DD_ADONET_ENABLED" value="true"/>
```

Reply:
Having `DD_ADONET_ENABLED` was just an example of how users can change the default values. All integration are already enabled by default. Feel free to remove the line entirely if it's confusing. Also, we realized recently that it's case-sensitive on Linux, so it should be `DD_AdoNet_ENABLED`.